src/auth/cognito/types/credentials.ts

```diff
@@ -0,0 +1,34 @@
+
+
+/**
+ * @category credentials 
+ */
+export type AWSCognitoAuthSession = {
+    isSignedIn: boolean,
+    pluginData: {
```

Comment:
If all of the fields of `pluginData` are optional, could `pluginData` itself be optional?